app/views/BusinessTypeView.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
```

Comment:
Can you please amend the spacing in this file similar to other views

Reply:
Formatted page as per other view pages.